.changeset/fresh-tables-clap.md

```diff
@@ -0,0 +1,75 @@
+---
+"@meilisearch/instant-meilisearch": minor
+---
+
+- Change the behavior of the `or` operator parameter on the [`RefinmentList`](https://www.algolia.com/doc/api-reference/widgets/refinement-list/js/) widget.
+
+⚠️ This impacts the facet distribution number
+
+## Drawback
+
+Currently, for each `facet value` selected in a different `facet` a separate request is made to Meilisearch. Plus an additional one containing all the facets filters. For example, if I select `Adventure` on genres and `Multiplayer` on players, one request is made with both facet filters, one with only `genres=Adventure` and one with only `players=Multiplayer`
```

Comment:
I can't suggest 🤔 

WDYT?
```md
Currently, for each `facet value` selected in a different `facet` a separate request is made to Meilisearch. Plus, an additional one containing all the facets filters. 
For example, if I select `Adventure` on genres and `Multiplayer` on players, one request is made with both facet filters, one with only `genres=Adventure` and one with only `players=Multiplayer`
```

Reply:
Just to be sure I saw the diff. You only added a coma after `Plus` ?

Reply:
yes!